javascript-functions/main.js

```diff
@@ -0,0 +1,33 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = (minutes * 60);
+  console.log('convertMinutesToSeconds:', seconds);
```

Comment:
you do not want to have any console logs in the functions. each of the functions should return a value, that value should be stored in a variable, and you should console log that variable. Make sure to check out the `gif` in the instructions!